layers/+lang/ocaml/packages.el

```diff
@@ -32,6 +33,34 @@
       :modes merlin-mode
       :variables merlin-completion-with-doc t)))
 
+(defun ocaml/init-dune ()
+  (use-package dune
+    :defer t
+    :init
+    (progn
+      (spacemacs/set-leader-keys-for-major-mode 'tuareg-mode
+        "tP" 'dune-promote
+        "tp" 'dune-runtest-and-promote)
+      (spacemacs/set-leader-keys-for-major-mode 'dune-mode
+        "il" 'dune-insert-library-form
+        "ie" 'dune-insert-executable-form
+        "ix" 'dune-insert-executables-form
+        "ir" 'dune-insert-rule-form
+        "ip" 'dune-insert-ocamllex-form
+        "iy" 'dune-insert-ocamlyacc-form
+        "im" 'dune-insert-menhir-form
+        "ia" 'dune-insert-alias-form
+        "ii" 'dune-insert-install-form
+        "ic" 'dune-insert-copyfiles-form
+        "it" 'dune-insert-tests-form
+        "iv" 'dune-insert-env-form
+        "id" 'dune-insert-ignored-subdirs-form
+        "tP" 'dune-promote
+        "tp" 'dune-runtest-and-promote
+        "cc" 'compile)
+      (spacemacs/declare-prefix-for-mode 'dune "mc" "compile/check")
+      (spacemacs/declare-prefix-for-mode 'dune "mi" "insert-form"))))
```

Comment:
I think the prefix for `t` is missing.

Reply:
Yep. Added declarations for both `dune-mode` and `tuareg-mode.